fawltydeps/extract_imports.py

```diff
@@ -172,19 +172,32 @@ def parse_dir(path: Path) -> Iterator[ParsedImport]:
             yield from parse_notebook_file(file, local_context=local_context)
 
 
-def parse_any_arg(arg: PathOrSpecial) -> Iterator[ParsedImport]:
+def parse_any_arg(
+    arg: PathOrSpecial, stdin: Optional[TextIO] = None
+) -> Iterator[ParsedImport]:
     """Interpret the given command-line argument and invoke a suitable parser.
 
     These cases are handled:
       - arg == "-": Read code from stdin and pass to parse_code()
       - arg refers to a file: Call parse_python_file() or parse_notebook_file()
       - arg refers to a dir: Call parse_dir()
 
-    Otherwise raise UnparseablePathException with a suitable error message.
+    Otherwise raise UnparseablePathException or StdInNotProvidedException
+    with a suitable error message.
```

Comment:
Nit: I don't think we lose any clarity if we omit this part of the commit message.
```suggestion
```